lib/minitest/hammer_coverage_plugin.rb

```diff
@@ -0,0 +1,26 @@
+require 'coverage_reporter'
+require "json"
+
+module Minitest
+  def self.plugin_hammer_coverage_options(opts, options)
+    opts.on "-c", "--coverage", "Generate coverage reports for API endpoints" do
+      options[:coverage] = true
+    end
+    opts.on "-v=VERSION", "--version=VERSION", "version" do |v|
+      options[:version] = v.split("=")[1]
+    end
+  end
+
+  def self.plugin_hammer_coverage_init(options)
```

Comment:
I'm not so familiar with `minitest`, so can you please explain where this method is being invoked from?..

Reply:
after loading the file  minitest/XXX_plugin.rb, Minitest will find and require that file using Gem.find_files, and then minitest will try to call plugin_XXX_init during startup